frontend/src/metabase-types/api/settings.ts

```diff
@@ -324,6 +324,7 @@ interface PublicSettings {
   "ldap-group-membership-filter": string;
   "loading-message": LoadingMessage;
   "map-tile-server-url": string;
+  "native-query-autocomplete-match-style": string;
```

Comment:
It's a [public setting](https://github.com/metabase/metabase/blob/master/src/metabase/api/database.clj#L620) so I think it can be in `PublicSettings`.